lib/active_model/serializable_resource.rb

```diff
@@ -18,6 +18,7 @@ def initialize(resource, options = {})
     # returns the contents of the block
     def self.serialize(resource, options = {})
       serializable_resource = SerializableResource.new(resource, options)
+      serializable_resource.notify_active_support
       if block_given?
         yield serializable_resource
       else
```

Comment:
move code into `initialize` as we're getting rid of `serialize` https://github.com/rails-api/active_model_serializers/pull/1129

event_name should be `active_model_serializers.serialize`, no?

This should also probably go in a logger like in https://github.com/rails/rails/blob/3e55fd755ac9c1d1f84f8dadd42112e2ee592a3a/activejob/lib/active_job/logging.rb#L19-L36 / https://github.com/rails/rails/blob/3e55fd755ac9c1d1f84f8dadd42112e2ee592a3a/activejob/lib/active_job/railtie.rb#L9-L10 and perhaps https://github.com/rails-api/active_model_serializers/pull/1089